python/lsst/ts/standardscripts/maintel/integration_tests/ccw_rotator_stress_test.py

```diff
@@ -0,0 +1,266 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["CcwRotatorStressTest"]
+
+import yaml
+import asyncio
+
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import Angle
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+np.random.seed(14)
+
+
+class CcwRotatorStressTest(salobj.BaseScript):
+    """Perform stress test between Camera Cable Wrap (CCW) and Rotator.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    * rotator_angle: before staring each sequence of random rotator angle.
+
+    * cycle: before each cycle of + and -.
+
+    * reset_rotator: after first test completes, before reseting the Rotator
+        position.
+
+    * delay: Before each "delay test".
+
+    **Details**
+
+    The test consists of:
+
+    * Start by slewing/tracking a circumpolar target with the rotator angle set
+      to be close to zero.
+    * Pick a configurable number of random rotator positions inside the limit.
+    * For each random rotator position P slew to +P, wait for the Rotator to
+      arrive at the position, let Rotator track for 15s and slew to -P. Repeat
+      this 5 times.
+    * Once done, reset to track a target with rotator angle close to zero.
+    * Send a demand to track a target at +80 degrees, but send a new target
+      in the other direction while the rotator is still slewing. Repeat this
+      test after a configurable series of time delay after slewing started.
+
+
+
+    Warning:
+    --------
+    This script requires to slew the telescope. Make sure the system is ready
+    to slew before running the script.
+
+    """
+
+    def __init__(self, index):
+        super().__init__(
+            index=index,
+            descr="Perform stress test between Camera Cable Wrap (CCW) and Rotator.",
+        )
+
+        self.mtcs = MTCS(
+            domain=self.domain, log=self.log, intended_usage=MTCSUsages.Slew
+        )
+
+        # How long to track for before each new slew (in seconds).
+        self.track_time = 15.0
+
+        # Configuration parameters.
+        self.n_rot_pos = 5
+        self.n_pos_repeat = 3
+        self.time_delays = np.array([1.0, 2.0, 5.0, 10.0])
+
+        # Ignore CSCs that are not part of the test. This will allow us to run
+        # the test if these CSCs are not enabled or being used for something
+        # else.
+        self.mtcs.check.mtaos = False
+        self.mtcs.check.mtm1m3 = False
+        self.mtcs.check.mtm2 = False
+        self.mtcs.check.hexapod_1 = False
+        self.mtcs.check.hexapod_2 = False
+        self.mtcs.check.dome = False
+        self.mtcs.check.mtdometrajectory = False
```

Comment:
Should these be set before `super().__init__`? It might be more obvious that they will have their desired effect if you do that.